### ISSUE TITLE
Generated projects do not work well with vscode-java

### ISSUE THREAD

**Issue by akaroml** [MicroShed/mp-starter-vscode-ext#14]
Generate a project using the starter and open it. The layout of the folder looks like this:
![image](https://user-images.githubusercontent.com/16755094/69846281-84cf2980-12ae-11ea-9d27-c2ba4f224d65.png)

There are two maven projects inside. But none could be loaded by vscode-java successfully.

It could be a vscode-java problem since the project could be loaded if I open `service-a` folder directly.

Would you like to work with redhat-developer/vscode-java to get this fixed?

**Comment by kathrynkodama** [comment]
Hi Rome, can you please provide detailed steps to reproduce this issue? When I generate a project with the MicroProfile Starter it seems to be picked up by vscode-java
![image](https://user-images.githubusercontent.com/26146482/69875692-c13d5e00-128b-11ea-8d5b-9383c605ff87.png)

**Comment by akaroml** [comment]
Thanks for the info. Let me try again to see if the issue persists.
